src/pj_factors.c

```diff
@@ -72,7 +72,7 @@ int pj_factors(LP lp, PJ *P, double h, struct FACTORS *fac) {
 
         /* convergence */
         if (!(fac->code & IS_ANAL_CONV)) {
-            fac->conv = - atan2(fac->der.y_l, fac->der.x_l);
+            fac->conv = - atan2(fac->der.x_p, fac->der.y_p);
```

Comment:
Please find attached the discussion and equations for meridian convergence.
[Meridian convergence in map projections.pdf](https://github.com/OSGeo/proj.4/files/1096686/Meridian.convergence.in.map.projections.pdf)
.